packages/gateway/src/features/registration/schema.graphql

```diff
@@ -15,6 +15,7 @@ enum RegStatus {
   REGISTERED
   CERTIFIED
   REJECTED
+  DOWNLOADED
```

Comment:
Is this a valid Registration Status? Like the bussinessStatus in a Task will never be `DOWNLOADED` because we are saving the `DOWNLOADED` status in extensions.

Reply:
this is not a valid bussinessStatus, but we need this here because while fetching from client, we need to send a valid status as well as DOWNLOADED using RegStatus.

Reply:
Maybe creating a separate type `HistoryStatus` that extends the `RegStatus` would have been better as it would keep the types separate?

Reply:
That was the initial intention, but could not figure out how to extend/union/join graphql enums